### PR TITLE
[Travis] Added Behat testing for PlatformUI

### PR DESCRIPTION
**Pull request by andrerom**
*Just for testing, will need to use correct profile and suite once that has been setup before we merge this.*

- [x] Get the tests to actually run, it is not correct that this is passing
- [x] Update for change to use Selenium
- [x] Update scenario and profile to be relevant tests for PlatformUI
- [ ] ezplatform tests passing: https://github.com/ezsystems/ezplatform/pull/46

### CODE REVIEW

**Comment by andrerom** [comment]
After 23 runs on travis this is now ready for tests, your turn @ezsystems/qa-team ;)

**Comment by pcardiga** [comment]
Roger :)

File: bin/travis/prepare_behat.sh [hunk 1]
```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
+
+# File for setting up system for behat testing, just like done in DemoBundle's .travis.yml
+
+# Install Apache before we add external repo for PHP 5.5 as it will conflict
+sudo apt-get update -q
+sudo apt-get install -q -y --force-yes apache2 libapache2-mod-fastcgi
+
+# Install PHP 5.5, apt-get package is php 5.3 since travis uses ancient Ubuntu
+sudo add-apt-repository -y ppa:ondrej/php5
```
Comment (dpobel) [inline bin/travis/prepare_behat.sh]: you should use (and complete) bin/travis/setupphp.sh

Comment (andrerom) [inline bin/travis/prepare_behat.sh]: I can do that, but then it will install fpm, mysqlnd, ..

**Comment by andrerom** [comment]
Ready for review, @dpobel @yannickroger @StephaneDiot @mhyndle

File: bin/travis/setupphp.sh [hunk 2]
```diff
@@ -2,4 +2,4 @@
 
 add-apt-repository -y ppa:ondrej/php5
 apt-get update -q
-apt-get install -y php5-cli php5-common php5-gd php5-xsl php5-intl
+sudo apt-get install -q -y --force-yes php5-cli php5-common php5-gd php5-xsl php5-intl
```
Comment (dpobel) [inline bin/travis/setupphp.sh]: you don't need to add sudo here as the script is already executed with sudo in .travil.yml and in prepare_behat.sh.

**Comment by dpobel** [comment]
besides the inline comment, +1

File: bin/travis/setupphp.sh (hunk 2 above)
Comment (andrerom) [inline bin/travis/setupphp.sh]: I did not use sudo in .travis.yml for this one, as I try to keep track if our sudo use for possibility to use travis container infrastructure. But I think they will fix that so if you want I can change it back to sudo in the yml file again.

**Comment by yannickroger** [comment]
+1

File: Features/Standard/platformui.feature [hunk 3]
```diff
@@ -1,28 +1,28 @@
 Feature: Basic PlatfomrUi interaction tests
```
Comment (pcardiga) [inline Features/Standard/platformui.feature]: Little Nitpick: There is a typo in "PlatfomrUi". Should be "PlatformUi" :)

Comment (andrerom) [inline Features/Standard/platformui.feature]: Not part of the PR,PR welcome :ship:

**Comment by andrerom** [comment]
And this is why I "hate"* Travis for functional testing :(


\** when something breaks there is a million things that could be the reason, and no clear way to debug*

**Comment by andrerom** [comment]
Closing as we got it working with ezci.
